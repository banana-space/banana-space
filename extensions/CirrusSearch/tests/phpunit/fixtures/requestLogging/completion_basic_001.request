{
	"type": "completion",
	"term": "main p"
}
