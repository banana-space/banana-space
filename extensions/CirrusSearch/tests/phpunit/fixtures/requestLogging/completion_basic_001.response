[{
    "responses": [
        {
            "took": 13,
            "timed_out": false,
            "_shards": {
                "total": 4,
                "successful": 4,
                "failed": 0
            },
            "hits": {
                "total": 0,
                "max_score": 0,
                "hits": []
            },
            "suggest": {
                "plain": [
                    {
                        "text": "main p",
                        "offset": 0,
                        "length": 6,
                        "options": [
                            {
                                "text": "Main Page",
                                "_index": "cirrustestwiki_titlesuggest_1485266182",
                                "_type": "titlesuggest",
                                "_id": "1t",
                                "_score": 5679327
                            }
                        ]
                    }
                ],
                "plain_fuzzy_1": [
                    {
                        "text": "main p",
                        "offset": 0,
                        "length": 6,
                        "options": [
                            {
                                "text": "Main Page",
                                "_index": "cirrustestwiki_titlesuggest_1485266182",
                                "_type": "titlesuggest",
                                "_id": "1t",
                                "_score": 29487956
                            }
                        ]
                    }
                ],
                "plain_stop": [
                    {
                        "text": "main p",
                        "offset": 0,
                        "length": 6,
                        "options": [
                            {
                                "text": "Main Page",
                                "_index": "cirrustestwiki_titlesuggest_1485266182",
                                "_type": "titlesuggest",
                                "_id": "1t",
                                "_score": 5679327
                            }
                        ]
                    }
                ],
                "plain_stop_fuzzy_1": [
                    {
                        "text": "main p",
                        "offset": 0,
                        "length": 6,
                        "options": [
                            {
                                "text": "Main Page",
                                "_index": "cirrustestwiki_titlesuggest_1485266182",
                                "_type": "titlesuggest",
                                "_id": "1t",
                                "_score": 23590364
                            }
                        ]
                    }
                ]
            }
        }
    ]
}]
