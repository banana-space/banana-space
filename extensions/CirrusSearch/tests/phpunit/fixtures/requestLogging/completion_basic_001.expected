[
    {
        "channel": "cirrussearch-request",
        "context": {
            "$schema": "\/mediawiki\/cirrussearch\/request\/0.0.1",
            "all_elasticsearch_requests_cached": false,
            "elasticsearch_requests": [
                {
                    "cached": false,
                    "hits": [
                        {
                            "index": "wiki_titlesuggest",
                            "page_id": 1,
                            "page_title": "Main Page",
                            "profile_name": "plain",
                            "score": 5679327
                        }
                    ],
                    "hits_offset": 0,
                    "hits_returned": 1,
                    "hits_total": 4,
                    "indices": [
                        "wiki_titlesuggest"
                    ],
                    "max_score": 5679327,
                    "query": "main p",
                    "query_type": "comp_suggest",
                    "search_time_ms": 0,
                    "suggestion_requested": false
                }
            ],
            "http": {
                "client_ip": "127.0.0.1",
                "has_cookies": false,
                "method": "GET"
            },
            "source": "cli"
        },
        "level": "debug",
        "message": ""
    },
    {
        "channel": "CirrusSearchRequests",
        "context": {
            "elasticTookMs": 0,
            "executor": "123456789",
            "hitsOffset": 0,
            "hitsReturned": 1,
            "hitsTotal": 4,
            "identity": "2c33c9c64851a02e2ba8143c6550a3b8",
            "index": "wiki_titlesuggest",
            "maxScore": 5679327,
            "query": "main p",
            "queryType": "comp_suggest",
            "source": "cli",
            "tookMs": 0
        },
        "level": "debug",
        "message": "{queryType} search for '{query}' against {index} took {tookMs} millis and {elasticTookMs} Elasticsearch millis. Found {hitsTotal} total results and returned {hitsReturned} of them starting at {hitsOffset}. Requested via {source} for {identity} by executor {executor}"
    }
]